<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>642</width>
    <height>432</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="FrontLabel">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>70</y>
      <width>31</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Front</string>
    </property>
   </widget>
   <widget class="QLabel" name="DownLabel">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>140</y>
      <width>31</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Down</string>
    </property>
   </widget>
   <widget class="QLabel" name="rightLabel">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>110</y>
      <width>31</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Right</string>
    </property>
   </widget>
   <widget class="QLabel" name="LeftLabel">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>110</y>
      <width>31</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Left</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>642</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
